ubuntu/ubuntu-18.x/ubuntu-18.04-hpc/install_utils.sh

```diff
@@ -48,7 +63,6 @@ apt-get -y install numactl \
                    libnl-route-3-dev \
                    libsecret-1-0 \
 		   ansible \
-		   python3-pip \
```

Comment:
@rafsalas19 , pls make sure this doesn't break Moneo.

Reply:
@moes1 @jithinjosepkl the file doesn't look right.  Maybe the Merge didn't go as planned?

Reply:
I don't think we need sudo here.

Reply:
We confirmed this doesn't break Moneo, and I cleaned up the duplicated lines.